src/lang/bg-BG.json

```diff
@@ -593,6 +593,7 @@
   "tab_wordcloud": "Wordcloud",
   "tab_mmr": "MMR",
   "tab_rankings": "Rankings",
+  "tab_drafts": "Draft",
```

Comment:
You only need to add to en-US file, the rest are automatically filled later

Reply:
Removed.